### ISSUE TITLE
Hot Reload fails to update XAML when Keyboard property is present

### ISSUE THREAD

**Issue by henda79** [dotnet/maui#24010]
### Description

This has been a long standing issue which I had noted with XF.

Hot Reload will not show changes to the XAML if the Keyboard property is used with arguments on controls such as Entry.

### Steps to Reproduce

                    <Entry.Keyboard>
                        <Keyboard x:FactoryMethod="Create">
                            <x:Arguments>
                                <KeyboardFlags>CapitalizeWord</KeyboardFlags>
                            </x:Arguments>
                        </Keyboard>
                    </Entry.Keyboard>

### Version with bug

9.0.0-preview.6.24327.7

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Android, I was *not* able test on other platforms

### Did you find any workaround?

No, I have to omit the Keyboard property in order to use Hot Reload

### Relevant log output

```shell
XAML: [Error] Hot Reload changes cannot be applied due to these errors in the XAML:
  TabView.xaml(44,24): The type "Keyboard" does not include any accessible constructors.
  TabView.xaml(44,34): The property "FactoryMethod" does not exist in the "http://schemas.microsoft.com/winfx/2009/xaml" namespace.
  TabView.xaml(91,28): The property "SelectedValue" does not have an accessible setter.
  TabView.xaml(106,28): The property "SelectedValue" does not have an accessible setter.
```

**Comment by etvorun** [comment]
This is an issue in VS. We are tracking it internally. For better visibility report this issue on [developercommunity.visualstudio.com](developercommunity.visualstudio.com)

**Comment by etvorun** [comment]
Fix for this issue will be available in the next VS Preview (17.12 Preview 2) and in the next VS Code pre-release.

**Comment by etvorun** [comment]
Fix for this issue will be available in the next VS Preview (17.12 Preview 2) and in the next VS Code pre-release.
